data/mods/gen6xy/moves.ts

```diff
@@ -0,0 +1,18 @@
+export const Moves: {[k: string]: ModdedMoveData} = {
+	dragonascent: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+	hyperspacefury: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+	originpulse: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+	precipiceblades: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
```

Comment:
all 4 of the pokemon (and their items/abilities in the case of the primals/mray) affiliated with these moves should be marked as Future (or unreleased in hoopa-confined's case, unbound would be Future/Nonexistent)